.github/workflows/docker.yml

```diff
@@ -43,7 +43,8 @@ jobs:
         with:
           # All available with python:3.11-slim are:
           # platforms: linux/386,linux/amd64,linux/arm/v5,linux/arm/v7,linux/arm64/v8,linux/ppc64le,linux/s390x
-          platforms: linux/amd64,linux/arm/v7,linux/arm64/v8
+          # But 32-bit binaries likely require compilation from source so stick with linux/amd64 and linux/arm64 for now
+          platforms: linux/amd64,linux/arm64
```

Comment:
`arm64` and not `arm64/v8`?

Reply:
I wish this was clearer, but from what I can tell from a comment on this stackoverflow answer - https://stackoverflow.com/a/70819313/6946787 is that arm64 is the same as arm64/v8. arm/v6 and arm/v7 however are 32 bit images.